src/popup/components/AccountItem.vue

```diff
@@ -5,12 +5,20 @@
     class="account-item"
     variant="muted"
   >
-    <Avatar
-      :address="address"
-      :name="name"
-      size="sm"
-    />
-
+    <div class="avatar-wrapper">
+      <Avatar
+        :address="address"
+        :name="name"
+        :class="{ avatar: protocol }"
```

Comment:
This class `avatar` makes no sense here. It adds space between the `Avatar` and `ProtocolIcon` when the `ProtocolIcon` is present. Why not add this margin to the `ProtocolIcon` without any checks?

Reply:
Protocol icon is positioned absolutely to overlap with the avatar - disabling the class would require to position the icon with negative margins

Reply:
Damn, now I understand what is happening :)
So what I suggest to use `has-protocol-icon` class on the `avatar-wrapper` which would add right padding instead of giving right margin on a child to make the wrapper wider.